src/algorithms/porep/stacked_drg_circuit.md

```diff
@@ -0,0 +1,161 @@
+## Stacked DRG: Offline PoRep Circuit Spec
```

Comment:
start these files with `#` -- the spec document should now shift headers correctly when grafting this header tree into the larger document.